.github/workflows/check-version.yml

```diff
@@ -0,0 +1,87 @@
+name: Check version
+
+on:
+  push:
+    branches:
+      - develop
+      - release/*
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  VERSION_FILE: ./package.json
+
+jobs:
+  check-version:
+    name: Check version
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      pull-requests: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Get current version
+        id: current_version
+        run: |
+          currentVersion=$(node -p -e "require('${{ env.VERSION_FILE }}').version")
+
+          echo "version=$currentVersion" >> $GITHUB_OUTPUT
+
+      - name: Get latest version
+        id: latest_version
```

Comment:
It could be a bit more descriptive
```suggestion
        id: get_latest_version
```

💡 I was unsure about the casing for step IDs and checked that it is indeed kebabcase as per the docs: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype. It could be clarified in https://nimblehq.co/compass/development/code-conventions/github-actions/#naming

Reply:
Fixed in https://github.com/nimblehq/infrastructure-templates/pull/237/commits/3b0ed365bfb509d6ebd0f30f4d99a58704f62951